challenge_2/go/erocs/find_singleton.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+)
+
+func FindSingleton(int_array []int) (found int) {
+	singles := make(map[int]int)
+	repeats := make(map[int]int)
```

Comment:
Any reason you can't just have one map of `map[int]int` and just loop over it at the end to find the (or a) single and return that? One less map allocation?